guides/source/active_record_postgresql.md

```diff
@@ -504,6 +504,11 @@ irb> device.id
 NOTE: `gen_random_uuid()` (from `pgcrypto`) is assumed if no `:default` option was
 passed to `create_table`.
 
+To use the Rails model generator to make a table with UUID as the primary key, pass `--primary-key-type=uuid` as a flag to the model generator.
+
+When building a foreign key that will reference this UUID, treat `uuid` as the native field type, for example:
+
+`rails generate model Case device_id:uuid`
```

Comment:
This needs a new line but also needs to be code-blocked with ruby like the above examples.

```suggestion
`rails generate model Case device_id:uuid`

```